components/widgets/separator/basic-separator.tsx

```diff
@@ -4,7 +4,7 @@
 import React from 'react';
 import './separator.scss';
 
-export default class BasicSeparator extends React.PureComponent<React.PropsWithChildren<{}>> {
+export default class BasicSeparator extends React.PureComponent<React.PropsWithChildren<any>> {
```

Comment:
Instead of using `React.PropsWithChildren` with an `any` in it, we could manually define our own Props type as `{children: React.ReactNode}`

Reply:
done